client/js/screens/AboutUs/AboutUs.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {View, Text, Image} from 'react-native';
+import styles from './styles';
+
+import AboutUsImage from '../../assets/artwork/adbust@.png';
+
+const AboutUs = () => {
```

Comment:
For best practice, your presentation components should accept arg `props` even when you are not using it in your component.